addons/payment_paypal/views/payment_provider_views.xml

```diff
@@ -10,13 +10,8 @@
                 <group attrs="{'invisible': [('code', '!=', 'paypal')]}">
                     <field name="paypal_email_account"
                            attrs="{'required':[('code', '=', 'paypal'), ('state', '!=', 'disabled')]}"/>
-                    <field name="paypal_seller_account" password="True"/>
-                    <!-- This field should no longer be used but is kept in debug mode for the time
-                         being, until we are sure that the verification protocol of IPN can be used
-                         for DPT notifications -->
-                    <field name="paypal_pdt_token" password="True"/>
-                    <field name="paypal_use_ipn"
-                           attrs="{'required':[('code', '=', 'paypal'), ('state', '!=', 'disabled')]}"/>
+                    <field name="paypal_pdt_token"  password="True"
+                           attrs="{'required': [('code', '=', 'paypal'), ('state', '!=', 'disabled')]}"/>
```

Comment:
Now that the PDT is required, we should be able to drop the `if not provider_sudo.paypal_pdt_token` check in the controller for the return from checkout.